pad_ui/PackDownloadPage.gd

```diff
@@ -0,0 +1,38 @@
+"""
+	Copyright 2020 Google LLC
+	
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+	
+		https://www.apache.org/licenses/LICENSE-2.0
+	
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+"""
+"""
+	Object used to render a list of PackDownloadDiv objects given an Array of packNames
+"""
+
+extends Node2D
+
+const pack_download_div_scene = preload("res://pad_ui/pack_download_div.tscn")
+const div_height = 90
+
+var pack_names
+var pad_manager
+
+func init(_pack_names : Array, _pad_manager : PlayAssetPackManager):
+	pack_names = _pack_names
+	pad_manager = _pad_manager
+	var div_count = 0
+	for pack_name in pack_names:
```

Comment:
Since you need the index to set the instance position, I would use a range() for loop instead:
````
for i in range(pack_names.size):
````

Reply:
Refactored this line using range(). I really which GDScript have enumerate() feature such as 
```
for idx, pack_name in enumerate(pack_names, start=0):
```